### ISSUE TITLE
Handling terminal resizes on Windows

### ISSUE THREAD

**Issue by aschey** [charmbracelet/bubbletea#538]
Since the PR to handle input records on Windows wasn't merged, would there be any interest in using a simple polling-based approach to handle resizes on Windows? Basically something like this:
```go
w, h, err := term.GetSize(int(f.Fd()))
for {
    time.Sleep(500 * time.Millisecond)
    newW, newH, err := term.GetSize(int(f.Fd()))
    if newW != w || newH != h {
        w = newW
        h = newH
        msgs <- WindowSizeMsg{w, h}
    }
}
```
Obviously not ideal, but I think it would be nice to support resizing in some capacity. Can make it disabled by default and marked as an unstable feature if necessary.

**Comment by muesli** [comment]
Personally I dislike such polling based solutions, as they always feel like a bit of a hack. Obviously the current situation isn't ideal and we need to some thing about it, but I'm sure there are proper ways to get notified of a resized console on Windows. Which PR are you referring to?

**Comment by aschey** [comment]
Agree that it's definitely a hack, but I don't know if there's another reliable way to do it besides handling input records. This: https://github.com/charmbracelet/bubbletea/pull/140 is the PR that implemented input records and would've enabled handling resizes. If there is still interest in handling input records, that would be the ideal way to solve this. But as mentioned in the comments, that would require a substantial change to the architecture around how the cancel reader now works.

**Comment by meowgorithm** [comment]
Just a note that this is now available in Windows courtesy #878. It's currently on `master` and will be included in the next release.
